libminifi/include/FlowFileRecord.h

```diff
@@ -108,6 +109,10 @@ class FlowFileRecord
 	 * Create a new flow record
 	 */
 	FlowFileRecord(std::map<std::string, std::string> attributes, ResourceClaim *claim = NULL);
+	/*!
+	 * Create a new flow record from repo flow event
+	 */
+	FlowFileRecord(FlowFileEventRecord *event);
```

Comment:
will use explicit, i am not using smart pointer here is that if you look at where we construct FlowFileRecord from FlowFileEventRecord, i am using a FlowFileEventRecord in the stack to. I want to avoid to create a new FlowFileEventRecord every time.